basis/listener/listener.factor

```diff
@@ -24,8 +24,13 @@ H{
 M: object prompt.
     nip prompt-style get-global format bl flush ;
 
+: with-ctrl-break ( quot -- )
+    enable-ctrl-break
+    [ disable-ctrl-break ] [ ] cleanup ; inline
+
 : parse-lines-interactive ( lines -- quot/f )
-    [ parse-lines ] with-compilation-unit ;
+    [ [ parse-lines ] with-ctrl-break
```

Comment:
OK, that explains it.

I usually have more branches than just the `master` to handle these things - like the `next` branch for the upcoming release, etc. But that, perhaps, comes with branch managing costs which I enjoy, but others may not like to waste their time on. Anyway, I'll leave this link here just in case: http://nvie.com/posts/a-successful-git-branching-model/

Ah, the 0.98 release... I remember, if I'm allowed to be a bit sentimental, I remember this quote by John Benediktsson as clearly as if it was uttered just on September 22nd, 2015: "Everything that's been committed since 0.97 (see the nightly builds) will be in 0.98 and our plan right now is to fix bugs and cleanup in preparation for a release in a week or two."

Good ol' days... : ))

Excuse me, friends, if I sound a bit frisky. I've been binging on "A bit of Fry and Laurie" lately, that explains my somewhat poetic mood.